docgen_processor/pom.xml

```diff
@@ -47,15 +47,13 @@
     </dependency>
     <dependency>
       <groupId>com.google.auto.service</groupId>
-      <artifactId>auto-service</artifactId>
-      <version>1.0-rc6</version>
-      <optional>true</optional>
+      <artifactId>auto-service-annotations</artifactId>
```

Comment:
This submodule is now missing annotation processing

Reply:
I'm confused. There's the annotation processing for compiling error prone, which seems to be ok because if there were an issue the compile should be failing with an error on a missing symbol for the annotation right?

Are you referring to a downstream effect due to how this piece fits into error prone's design, such that error prone plugin users will have issues with annotations? Like is there a way I can produce a failure somehow to understand the consequence or how this is incorrect?

Reply:
docgen submodules are used to generate the documentation for the website.
IIUC, they're used when building while enabling the `run-annotation-processor` profile: docgen_processor is then applied to core to generate a txt file, and docgen is executed with that txt file as input to generate the markdown files.
Without the auto-service processor here, the txt file won't be generated.

Reply:
This comment was super helpful. I found util/generate-latest-docs.sh and was able to reproduce the issue, and this latest push should fix this and the other comments raised.